pyproject.toml

```diff
@@ -149,7 +149,7 @@ pytest-timeout = ">=1.0.0"
 pytest-xdist = ">=1.14"
 "repoze.debug" = ">=1.0.2"
 
-wheel = ">=0.29.0"  # needed for distribution but not any particular version
+wheel = "0.37.1"
```

Comment:
Does this have any real impact since it is in dev-dependencies?

Reply:
Well, even in dev dependencies, the solver will complain if there is no way to reconcile the set of claimed versions. It wants to resolve this as lock time, not at load time.